generator/graphql-kotlin-schema-generator/src/main/kotlin/com/expediagroup/graphql/generator/internal/types/generateArgument.kt

```diff
@@ -51,11 +52,16 @@ internal fun generateArgument(generator: SchemaGenerator, parameter: KParameter)
     val graphQLType = generateGraphQLType(generator = generator, type = unwrappedType, typeInfo)
 
     // Deprecation of arguments is currently unsupported: https://youtrack.jetbrains.com/issue/KT-25643
```

Comment:
can you update the comment to `Deprecation of arguments through @Deprecated is currently unsupported: https://youtrack.jetbrains.com/issue/KT-25643`